docs/examples.rst

```diff
@@ -293,3 +293,45 @@ prettyprints the single dimension dict:
             "section_pk": 707,
         },
     }
+
+How to Data Upload
+^^^^^^^^^^^^^^^^^^^
+
+Currently the API supports the ability to search for data groups one owns, check the processing status, and upload valid DataFrames. One must still upload data through the UI - specifying the schema. The API is meant to programmatically refresh the data.
+
+..  code::
+
+    for data_group in dm.get_data_groups():
+        print(data_group)
+
+Alternatively, one can fetch a data group by its ID and view it's status:
+
+..  code::
+
+    dg = dm.get_data_group_by_id(1012)
+    dg.get_current_status()
+
+To view the columns of the data group, and hence the schema, to verify the type of data we wish to reupload:
+
+..  code::
+
+    dg.columns
+
+To refresh the data, call `start_data_refresh` with a valid `pandas.DataFrame` object that matches the schema of the data group.
+
+..  code::
+
+    df = pandas.DataFrame({
+        'Start_Date': ['2019-01-01'],
+        'end date': ['2019-01-02'],
+        'dummy data 1': [1],
+        'dummy data_2': [1],
+        'Ticker': ['AAP'],
+        ...
+    })
+    dg.start_data_refresh(df)
+    dg.get_current_status()
+
+One will notice the `status` of the data group object change.
```

Comment:
maybe this?
The `status` of the data group object will change to reflect the latest status

Reply:
thx

Reply:
sorry this actually sounds like the `data_group.status` will change without calling `dg.get_current_status()` again. but that's not the case right?

Reply:
i think that is discussed in the docs of the class - hence why i was vague and didn't specify exactly what to call in code

Reply:
i see. given the context it should be clear then. also the function is well named